filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```diff
@@ -568,8 +568,18 @@ filebeat.inputs:
   # You may wish to have separate inputs for each service. You can use
   # include_matches to specify a list of filter expressions that are
   # applied as a logical OR. You may specify filter
-  #include_matches:
-    #- _SYSTEMD_UNIT=foo.service
+  #include_matches.or:
+    #- equals:
+      #- _SYSTEMD_UNIT=foo.service
+
+  # List of syslog identifiers
+  #identifiers: ["audit"]
+
+  # List of units
+  #units: ["docker.service"]
+
+  # If kernel entries are need to be forwarded if units are configured
+  #kernel: false
```

Comment:
Yes, it is. The options I have added are coming from the community Journalbeat. I implemented filtering the same way so people who are migrating from that to the Elastic solution does not have to change anything in their configuration. But given that we changed the configuration a lot in the last few years, it is probably no longer matters because they cannot avoid changing configuration.